coderd/database/migrations/000022_workspace_builds_with_initiaitor_view.up.sql

```diff
@@ -0,0 +1,7 @@
+-- This view adds the initiator name to the query for UI purposes.
+-- Showing the initiator user ID is not very friendly.
+CREATE VIEW workspace_build_with_initiator AS
+-- If the user is nil, just use 'unknown' for now.
+SELECT workspace_builds.*, coalesce(users.username, 'unknown') AS initiator_username
+FROM workspace_builds
+		 LEFT JOIN users ON workspace_builds.initiator_id = users.id;
```

Comment:
Will this require adding a migration every time we want to change the query?

Reply:
@kylecarbs yes. Another migration will drop the old view and make a new one.

If we change the columns, it will be required too as the `dump.sql` expands `*`. I just realized that :thinking:

Reply:
Unless we implement something else to "migrate" views.

A view is a saved query on the database. The alternative is to use temporary views which only last for a single "session". So this alternative thing to "migrate" views could be a set of temporary views we initiate when we open a SQL connection. (maybe, brainstorming here, I think that would work)